doc/fgpu.design.rst

```diff
@@ -734,3 +724,7 @@ The work group size, subgroup size and coarsening factor can all affect
 performance significantly, and not always in obvious ways. It will likely be
 necessary to implement autotuning to get optimal results across a range of
 problem parameters and hardware devices, but this has not yet been done.
+
+.. rubric:: Footnotes
+
+.. [#] Going back to the original definition for the DFT we saw the complex exponential :math:`e^{\frac{-2\pi j}{2N}\cdot ik}` has a variable :math:`k` where :math:`k` represents the frequency component under computation for the input sequence :math:`x_i`. If :math:`k` is reversed (i.e. negative) the complex exponential changes to :math:`e^{\frac{2\pi j}{2n}\cdot i(k)}` as the negative in :math:`-k` multiplies out.
```

Comment:
```suggestion
.. [#] Going back to the original definition for the DFT we saw the complex exponential :math:`e^{\frac{-2\pi j}{2N}\cdot ik}` has a variable :math:`k` where :math:`k` represents the frequency component under computation for the input sequence :math:`x_i.` If :math:`k` is reversed (i.e. negative) the complex exponential changes to :math:`e^{\frac{2\pi j}{2n}\cdot i(k)}` as the negative in :math:`-k` multiplies out.
```

At the moment the full stop is getting wrapped onto the next line in the HTML output. Putting it inside the math prevents the line being broken in the wrong place.